src/frontend/context/BrevModulContext.ts

```diff
@@ -189,9 +225,10 @@ const [BrevModulProvider, useBrevModul] = createUseContext(() => {
             brevmal,
             dokumenter,
             fritekster,
-            barnaMedOpplysninger,
+            barnMedDeltBosted,
+            barnBrevetGjelder,
             avtalerOmDeltBostedPerBarn,
-            datoAvtale: datoAvtale,
+            datoAvtale,
```

Comment:
Har du testet å sende et brev som har datoAvtale feltet etter denne endringen? Det var jeg som la til dette en gang i tiden. Husker ikke helt hvorfor, men føler det var noe som ikke funket når jeg skrev det på denne måten 🤔

Reply:
Kan teste også, bare for å ha gjort det 😇

Reply:
Testet OK mot preprod 💌